src/main/java/io/reactivex/Maybe.java

```diff
@@ -865,6 +867,51 @@
         return RxJavaPlugins.onAssembly(new MaybeFromRunnable<T>(run));
     }
 
+    /**
+     * Returns a {@link Maybe} that invokes the given {@link Supplier} for each individual {@link MaybeObserver} that
+     * subscribes and emits the resulting non-null item via {@code onSuccess} while
+     * considering a {@code null} result from the {@code Supplier} as indication for valueless completion
+     * via {@code onComplete}.
+     * <p>
+     * This operator allows you to defer the execution of the given {@code Supplier} until a {@code MaybeObserver}
+     * subscribes to the  returned {@link Maybe}. In other terms, this source operator evaluates the given
+     * {@code Callable} "lazily".
```

Comment:
Supplier.

Reply:
Thanks corrected this and other mistakes.